.eslintrc.js

```diff
@@ -50,6 +50,8 @@ module.exports = {
         "content-src/asrouter/templates/OnboardingMessage/**",
         "content-src/asrouter/templates/Trailhead/**",
         "content-src/components/TopSites/**",
+        "content-src/components/MoreRecommendations/MoreRecommendations.jsx",
+        "content-src/components/CollapsibleSection/CollapsibleSection.jsx"
```

Comment:
With updated PR, we can remove  "content-src/components/CollapsibleSection/CollapsibleSection.jsx"

Reply:
never mind, noticed getFormattedMessage refactor PR needs eslint to ignore CollapsibleSection